scss/main.scss

```diff
@@ -287,23 +287,55 @@ h1,h2,h3 {
   margin-bottom: 2rem;
   padding-left: 2rem;
   padding-right: 2rem;
+  p.text-center {
+    margin-bottom: 48px;
+  }
 }
 
 #meetupLink {
   color: $cfa-blue;
 }
 
+.meetup-content {
+  margin-bottom: 1.5rem;
+  > * {
+  margin-bottom: 1rem;
+  }
+}
+
+
+
 .rsvp-stuff {
   margin-top: 1rem;
+  p {
+    font-family: $font-family-alt;
+    font-weight: 500;
+    color: $darker-grey;
+    font-size: .875rem; 
+  }
 }
 
 .attendees {
   border-bottom: 2rem solid transparent;
 }
 
 .attend-button {
+  font-family: $font-family-alt;
+  font-weight: 500;
```

Comment:
Seems like this should be 600 instead

<img width="615" alt="Screen Shot 2021-02-10 at 9 49 51 PM" src="https://user-images.githubusercontent.com/9326234/107600693-18f28b80-6bea-11eb-8edd-bfb9651fc2cb.png">

Reply:
Thank you! I'm just waiting on joseph for the font weight, the trello and figma appear to be in disagreement.